src/entities/mellow/mellowLpRouter.ts

```diff
@@ -821,6 +832,124 @@ class MellowLpRouter {
 
     return this.canManageVaultPositions[vaultIndex];
   };
+
+  submitBatch = async (): Promise<ContractReceipt> => {
```

Comment:
I would rename this `submitAllBatchesForFee`

Reply:
renamed [a09f7f2](https://github.com/Voltz-Protocol/v1-sdk/pull/164/commits/a09f7f220a9a2e4d81d125cf7c6c0b89fb49a6f5)